### PR TITLE
Components of a Software Bill Of Materials (SBOM)

### PR DESCRIPTION
**Pull request by quozl**
A Software bill of materials[^1] (SBOM) in prose form was constructed by looking through the source code[^2] for various means of dependence[^3].

[^1]: [Software bill of materials - Wikipedia](https://en.wikipedia.org/wiki/Software_bill_of_materials)
[^2]: Sugar, Sugar Artwork, Sugar Toolkit, Sugar Datastore, and the Fructose activities Browse, Calculate, Chat, Image Viewer, Jukebox, Log, Pippy, Read, Terminal, Turtle Art, and Write.
[^3]: Python imports, C #includes, and fork and exec code patterns.

### CODE REVIEW

File: docs/components.md [hunk 1]
```diff
@@ -7,9 +7,9 @@
 | Component | Description | Repository |
 | :-------- | :---------- | :--------- |
 | Sugar | Activity Menu, Journal, Network View and Control Panel | https://github.com/sugarlabs/sugar |
-| Toolkit | Activity, Journal, and Plaform library | https://github.com/sugarlabs/sugar-toolkit-gtk3 |
-| Datastore | Journal Storage API | https://github.com/sugarlabs/sugar-datastore |
-| Artwork | Icons, Themes and Cursors | https://github.com/sugarlabs/sugar-artwork |
+| Sugar Toolkit | Activity, Journal, and Plaform library | https://github.com/sugarlabs/sugar-toolkit-gtk3 |
+| Sugar Datastore | Journal Storage API | https://github.com/sugarlabs/sugar-datastore |
```
Comment (chimosky) [inline docs/components.md:11]: I think this should be `Sugar Toolkit Gtk3` seeing as we have an existing `sugar-toolkit`.

**Review by chimosky — CHANGES_REQUESTED** [review]

**Comment by chimosky** [comment]
Reviewed a65ef0e, thanks for putting this together.

File: docs/components.md (hunk 1 above)
Comment (quozl) [reply docs/components.md:11]: https://github.com/sugarlabs/sugar/pull/954/commits/7fbc94e438208dc23ed27f6a9c33d5fa5204ec94 adds this change, but I'm not sure it helps, and it ties us to a release version range.

**Comment by chimosky** [comment]
Reviewed, looks good. Thanks!

**Pull request merged**